dts/msm8226/msm8926-v1-mtp.dts

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	compatible = "qcom,msm8926-mtp", "qcom,msm8926";
```

Comment:
Not sure if we really care about the to level compatible as I think the dtbtool in this lk tree is not smart enough to properly generate id's for this one (it can for some files so we use compatible instead of id-s in those).

Reply:
I based this off msm8916-mtp dtb which has the top level compatible

Reply:
msm8916-mtp has no top-level board/msm-id's though as they're autogenerated from that compatible :)

Reply:
Ohhh now I see..